jazz_deployments/index.js

```diff
@@ -158,13 +158,15 @@ function handler (event, context, cb) {
 		.catch((error) => {
 			if (error.result === "inputError") {
 				return cb(JSON.stringify(errorHandler.throwInputValidationError(error.message)));
```

Comment:
why changes in jazz_deployments? 
if some merge issue( please update to the latest pull from V1.5.2